src/index.ts

```diff
@@ -10,26 +19,75 @@ export async function sync(name: string, from: Record<string, string>, to: Recor
   // fullMetadata may needed to obtain the repository property in manifest
   srcOptions.fullMetadata = true
   const srcPackument = await getPackument(name, srcOptions)
-  const srcVersions = srcPackument ? Object.keys(srcPackument.versions) : []
-  console.debug('Source versions', srcVersions)
+  const srcVersionsRaw = srcPackument ? Object.keys(srcPackument.versions) : []
+
+  console.debug('Source versions', srcVersionsRaw)
+
+  let srcVersions = srcVersionsRaw;
+
+  if (srcPackument && latestOnly) {
+    srcVersions = [srcPackument['dist-tags'].latest]
```

Comment:
Codacy has a fix for the issue: [Replace `'dist-tags'].latest]` with `"dist-tags"].latest];`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
    srcVersions = [srcPackument["dist-tags"].latest];
```